src/Datepicker.js

```diff
@@ -68,6 +73,24 @@ export default class Datepicker extends Component {
     })
   }
 
+  handleChange(e) {
```

Comment:
Since you've change this to a controlled component, what happens if I pass in a `value` prop for the input after the component has been mounted?

Reply:
You wouldn't be able to pass down a new startDate (I assume that's what you mean by a `value` prop) once the component mounts. However, as best I can tell, this is consistent with the current date-picker functionality.